README.md

```diff
@@ -11,39 +13,53 @@
 ![GitHub contributors](https://img.shields.io/github/contributors/DXHeroes/dx-scanner)
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg)](#contributors-)
 [![NPM](https://img.shields.io/npm/l/dx-scanner)](LICENSE)
-[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
 ![TypeScript](https://img.shields.io/badge/%3C%2F%3E-Typescript-blue)
 
+---
+
+## What is DX Scanner?
+
 DX Scanner is an open source CLI tool that allows you to “measure” Developer Experience directly based on your source code. DX Scanner recommends practices that can help you with improving your product development. You can fix some problems automatically with just one command.
 
+### What language is supported?
+
+Language | Supported
+------------ | -------------
+JavaScript/TypeScript | ✅
+Java/Kotlin | ✅
+Python | ✅
+PHP | 🚧
+C++ | 🚧
+C# | 🚧
+Ruby | 🚧
+
+## Table of Contents
+
 <!-- toc -->
+* [Supported version control systems](#Which-version-control-system-can-you-use)
 * [Getting Started](#Getting-Started)
   * [Installation](#Installation)
   * [Usage](#Usage)
     * [Commands](#Commands)
-    * [Options for dx-scanner run`](#options-for-dx-scanner-run)
+    * [Options for dx-scanner run](#options-for-dx-scanner-run)
     * [Auto-fixer](#Auto-fixer)
 * [Supported Languages](#What-language-is-supported)
 * [Configuration](#Configuration-⚙️)
   * [Practices](#Practices)
   * [GitHub CI Action](#GitHub-Ci-Action)
 <!-- tocstop -->
+### Which version control system can you use?
 
-![DX Scanner Demo](./demo.svg)
+Sure you can use `GitHub`. You can also use `GitLab` and `Bitbucket` no matter if it is public or private repo!
 
+examples:
+```
+dxs run https://github.com/DXHeroes/dx-scanner
 
-## What language is supported?
-
-Language | Supported
------------- | -------------
-JavaScript/TypeScript | ✅
-Java/Kotlin | ✅
-Python | ✅
-PHP | 🚧
-C++ | 🚧
-C# | 🚧
-Ruby | 🚧
+dxs run https://gitlab.com/gitlab-org/gitlab
 
+dxs run https://bitbucket.org/atlassian/bitbucketjs
```

Comment:
Choose some smaller repo than these big ones. The user could copy&paste this row and try it and we should offer some faster scans.